lib/blemulator/simulation/simulated_peripheral.dart

```diff
@@ -135,4 +135,6 @@ abstract class SimulatedPeripheral {
     }
     return targetCharacteristic;
   }
+
+  int rssi() => scanInfo.rssi;
```

Comment:
Lets make it return Future<int> as this operation should be asynchronous

Reply:
why should it be asynchronous?

Reply:
In standard BT stack, obtaining actual rssi from device is an asynchronous operation because its being evaluated again (with no matter what was the rssi value during scanning process).